README.md

```diff
@@ -12,4 +12,52 @@ Binary package
 
 ## Usage
 
-*TODO: Add instructions for usage*
+```
+$ hab pkg install core/gawk
```

Comment:
Can't this just be one command e.g. 

```
[1][default:/src:0]# hab pkg install core/gawk --binlink
» Installing core/gawk
☁ Determining latest version of core/gawk in the 'stable' channel
→ Using core/gawk/5.0.1/20200305233704
★ Install of core/gawk/5.0.1/20200305233704 complete with 0 new packages installed.
» Binlinking awk from core/gawk/5.0.1/20200305233704 into /bin
★ Binlinked awk from core/gawk/5.0.1/20200305233704 to /bin/awk
» Binlinking gawk from core/gawk/5.0.1/20200305233704 into /bin
★ Binlinked gawk from core/gawk/5.0.1/20200305233704 to /bin/gawk
» Binlinking gawk-5.0.1 from core/gawk/5.0.1/20200305233704 into /bin
★ Binlinked gawk-5.0.1 from core/gawk/5.0.1/20200305233704 to /bin/gawk-5.0.1
[2][default:/src:0]# gawk --help
```

Reply:
@skpaterson:  Updated and squashed the commits.